apb.yml

```diff
@@ -9,7 +9,7 @@ tags:
 metadata:
   displayName: Metrics
   imageUrl: "https://avatars1.githubusercontent.com/u/3380462?s=200&v=4"
-  documentationUrl: "https://docs.aerogear.org/aerogear/latest/metrics/index.html"
+  documentationUrl: "https://docs.aerogear.org/external/apb/metrics.html"
   providerDisplayName: "Red Hat, Inc."
   sdk-docs-android: "https://docs.aerogear.org/aerogear/latest/metrics/index.html"
   sdk-docs-cordova: "https://docs.aerogear.org/aerogear/latest/metrics/index.html"
```

Comment:
what about those?

Also I think we need to run `apb prepare` to regenerate the dockerfile.

Reply:
@wei-lee after searching the the origin-web-console fork there is no occurrence of these variables. The are also not included in other apbs. I think we can remove them.

Reply:
@maleck13 can you confirm that those variables can be removed or not?